back/routes/api.php

```diff
@@ -110,6 +110,13 @@
     Route::get('invoices/{invoice_id}/download', 'InvoiceController@download')
         ->name('invoices.download');
 
+    // Message ressource routes
+    Route::apiResource('messages', 'MessageController', ['parameters' => ['messages' => 'message_id']])->except(['store']);
+    Route::get('messages', 'MessageController@index')->name('message.index');
+    Route::post('messages', 'MessageController@store')->name('message.store');
+    Route::put('messages', 'MessageController@update')->name('message.update');
+    Route::delete('messages', 'MessageController@destroy')->name('message.destroy');
```

Comment:
Là je pense que j'ai pas du voir l'erreur mais y'a 2 points à corriger : 
- Il faut que tu corriges ça : 
```suggestion
    // Message ressource routes
    Route::apiResource('messages', 'MessageController', ['parameters' => ['messages' => 'message_id']]);
```
En fait `Route::apiResource` te permets de générer les routes _index_, _show_, _create_, _update_ et _destroy_ donc tu n'as pas besoin de les redéclarer manuellement ensuite
- Et ensuite il faut que tu rajoutes la route PATCH /messages/{message_id}/activation parce que pour le moment on ne peut pas utiliser le webservice n'est pas accessible